drools-game-engine-horserace-kjar/src/main/resources/META-INF/kmodule.xml

```diff
@@ -0,0 +1,7 @@
+<kmodule xmlns="http://jboss.org/kie/6.0.0/kmodule" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <!--
+    <kbase name="globalKBase" includes="coreGameKBase" packages="rules.game.global" default="false"></kbase>
```

Comment:
We should mark this as required, because the coreGameKBase defined inside the drools-game-engine-core-kjar defines the generic rules for every game.